src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -44,6 +44,7 @@ public function validate($value, Constraint $constraint)
 
         if (!$this->getExpressionLanguage()->evaluate($constraint->expression, $variables)) {
             $this->context->buildViolation($constraint->message)
+                ->atPath($constraint->errorPath)
```

Comment:
I guess this should be called only when `$constraint->errorPath !== null`

Reply:
or cast to string which was originally added. So what would be the best way? 

cc/ @nicolas-grekas